docs/griptape-cloud/structures/create-structure.md

```diff
@@ -0,0 +1,16 @@
+# Structures
+
+Structures are the primary building block in the Griptape.
```

Comment:
That's a very broad assertion. I think we should say the primary building block for execution, or running Agents, or accessing LLMs, or something.

Reply:
🤔 it's part of our docs for the Framework. [Top level of Core Components](https://github.com/griptape-ai/griptape?tab=readme-ov-file#%EF%B8%8F-structures), first thing in [Getting Started](https://docs.griptape.ai/latest/griptape-framework/#build-a-simple-agent).

Reply:
Tweaked wording.